app/views/layouts/application.html.erb

```diff
@@ -6,13 +6,27 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```

Comment:
Great job adding your CDN assets to a partial! This helps keep your layout file clean :)